opsgenie/resource_opsgenie_team.go

```diff
@@ -164,7 +164,7 @@ func resourceOpsGenieTeamUpdate(d *schema.ResourceData, meta interface{}) error
 		Description: description,
 	}
 
-	if len(d.Get("member").([]interface{})) > 0 && !d.Get("ignore_members").(bool) {
```

Comment:
I think we also need this change for the `resourceOpsgenieTeamCreate` function, no?

Reply:
Thanks a lot for spotting that! :man_facepalming:  (I admit I only tested the update of teams, not the creation :man_facepalming: )

I made another fix here https://github.com/opsgenie/terraform-provider-opsgenie/pull/321 and checked with the acceptance tests (should have done that since the beginning) and with a manual tests as well.